ONSdigital/eq-survey-runner
# eq-181 mci patterns

This PR integrates the styling from the first prototype, documented in the pattern library and with an example survey layout.
## Changes
- Added temporary route for mci prototype at `/mci/`.
- Schema for prototype questions is loaded from `data.json`.
- Base layout created for surveys including header, basic page structure, etc.
- Partials created for each field type.
- Styling for each type of field and associated inputs.
- Updated forms page in pattern library with new input types.
- Added a 'questions' section to pattern lib.
- Some enhancements for legacy browsers.
- Refined gulp tasks for copying javascript files to static folder.
## Testing
- Do the usual `npm` stuff.
- Visit `/mci/` and confirm page exists, etc.
- Visit pattern library and confirm forms/questions pages exist, etc.
